src/style.css

```diff
@@ -0,0 +1,57 @@
+body {
+  margin: 0;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  position: relative;
+  display: flex;
+  text-align: center;
+  font-family: Roboto;
```

Comment:
Which family will be displaying if google fonts not responding?
Please, add additional common families like `Arial` or `sens-serif`.